templates/sso74-image-stream.json

```diff
@@ -13,165 +13,53 @@
             "kind": "ImageStream",
             "apiVersion": "v1",
             "metadata": {
-                "name": "redhat-sso-cd-openshift",
+                "name": "redhat-sso74-openshift",
                 "annotations": {
-                    "description": "Red Hat Single Sign-On Continuous Delivery",
-                    "openshift.io/display-name": "Red Hat Single Sign-On Continuous Delivery",
+                    "description": "Red Hat Single Sign-On 7.4 on OpenJDK",
+                    "openshift.io/display-name": "Red Hat Single Sign-On 7.4 on OpenJDK",
                     "openshift.io/provider-display-name": "Red Hat, Inc.",
-                    "version": "6"
+                    "version": "7.4.0.GA"
                 }
             },
             "labels": {
-                "rhsso": "6"
+                "rhsso": "7.4.0.GA"
             },
             "spec": {
                 "tags": [
                     {
                         "name": "latest",
                         "annotations": {
-                            "description": "Latest build of Red Hat Single Sign-On Continuous Delivery Tech Preview image",
+                            "description": "Latest build of Red Hat Single Sign-On 7.4 OpenJDK image",
                             "iconClass": "icon-sso",
                             "tags": "sso,keycloak,redhat,hidden",
                             "supports": "sso:7.4",
                             "version": "latest",
-                            "openshift.io/display-name": "Red Hat Single Sign-On Continuous Delivery (Tech Preview)"
+                            "openshift.io/display-name": "Red Hat Single Sign-On 7.4 on OpenJDK"
                         },
                         "referencePolicy": {
                             "type": "Local"
                         },
                         "from": {
                             "kind": "DockerImage",
-                            "name": "registry.redhat.io/redhat-sso-cd-tech-preview/sso-cd-openshift:latest"
+                            "name": "registry.redhat.io/redhat-sso-74/redhat-sso74-openshift-rhel8:1.0"
```

Comment:
Two times `redhat` here. Proper form is `registry.redhat.io/redhat-sso-7/sso74-openshift-rhel8:1.0`

Reply:
Looking at 73 branch there is only `registry.redhat.io/redhat-sso-7/sso73-openshift:1.0`, should there be this `-rhel8` suffix? It was in CD releases but not sure it should stay there.

Reply:
Yes, the `-rhel8` suffix should be there (it's not present in RH-SSO 7.3 form yet, because it wasn't needed yet). But as the count of base image raises, it's needed to differentiate them.

Reply:
Thank you @iankko, I fixed it